python/pylibcudf/pylibcudf/tests/test_datetime.py

```diff
@@ -12,19 +10,52 @@
 @pytest.fixture
 def column(has_nulls):
     values = [
-        datetime.date(1999, 1, 1),
-        datetime.date(2024, 10, 12),
-        datetime.date(1, 1, 1),
-        datetime.date(9999, 1, 1),
+        pa.scalar(1694004645123456789, pa.timestamp("ns")),
+        pa.scalar(1544024645123456789, pa.timestamp("ns")),
+        pa.scalar(1682342345346235434, pa.timestamp("ns")),
+        pa.scalar(1445624625623452452, pa.timestamp("ns")),
```

Comment:
Is there a reason we can't use conversion from datetime? It's also not obvious to me that these are the same timepoints as before.